### DISCUSSION TITLE
Adjust header font size to match main text

### DISCUSSION THREAD

**Discussion by maarten1C96** [PDConSec/vsc-print#271] [Ideas]
The document header font size in the generated HTML (in my webbrowser) is uncomfortably larger than the main text, of which I believe results in a visually jarring effect and inefficient use of paper space. Wouldn't it be better to include an option in the settings to manually adjust this header font size?

**Comment by PeterWone** [comment]
Crikey this one slipped under the radar. 

Good idea, I'll see what I can do.

The problem with taking user settings and embedding them into the generated page is that it means you have to embed a stylesheet which always takes precedence over external CSS. As a result I have to use a generated external stylesheet, which is more complicated to implement. But I'm pretty sure I have one of those going on somewhere so I should be able to base it on that.

Using direct settings for some things and stylesheets for others also makes it more complicated for people to understand, but you can say that about any decently versatile tool so I'm taking the idea seriously.
